config.ml

```diff
@@ -27,7 +27,7 @@ let packages = [
   package "omd" ;
   package ~min:"4.0.0" "tyxml";
   package "ptime";
-  package ~max:"0.4.0" "decompress";
+  package ~max:"0.5.0" "decompress";
```

Comment:
What's the current release, i think we need a `min` constraint here instead (or in addition to) `max`